wercker.yml

```diff
@@ -17,12 +20,4 @@ build:
             code: |
                 export RETHINKDB_HOST=$WERCKER_RETHINKDB_HOST
                 tox
-deploy:
```

Comment:
Good that you were cautious. Actually it can't upload anything; there are some variables defined in wercker that you don't have (the PyPI login). Also, it doesn't automatically upload to PyPI, someone has to explicitly press the 'Deploy' button after a successful build for this to happen.